test/e2e/framework.go

```diff
@@ -627,6 +765,10 @@ func randPodName(prefix string) string {
 	return prefix + randSeq(podNameSuffixLength)
 }
 
+func randSvcName(prefix string) string {
```

Comment:
It looks like not necessary to have one random name functions for each object?

Reply:
Yes, use randName() instead.